# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/sirkull/pyzor/makefile.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -urN pyzor-0.4.0.orig/Makefile pyzor-0.4.0/Makefile
--- pyzor-0.4.0.orig/Makefile	1969-12-31 19:00:00.000000000 -0500
+++ pyzor-0.4.0/Makefile	2003-06-25 13:29:43.000000000 -0400
@@ -0,0 +1,4 @@
+all:
+	python setup.py build
+install:
+	python setup.py install
